### PR TITLE
pipewire: update to 0.3.25

### LINKED ISSUE #29655: pipewire: can't load api.bluez5.enum.dbus: No such file or directory
I'm opening this one because I realized our packaging is wrong wrt pipewire and bluetooth. This error should not be there.

This is upstream default policy:
```
    with-pulseaudio = [
        with-audio
        bluez5
        restore-stream
        streams-follow-default
    ]
```
which means: load the bluetooth plugin when pulseaudio functionality is installed, and we install pulseaudio functionality by default.

Two possibilities:

1) Stay with upstream policy and add `libspa-bluetooth` to `depends`. This however adds unwanted dependencies for users not using bluetooth. And there is no going back once you do this.

2) Change the policy (it should be fine as a distro) into:
```
    with-bluetooth = [
        bluez5
    ]
    with-pulseaudio = [
        with-audio
        restore-stream
        streams-follow-default
    ]
```
And ship the `with-bluetooth` file in `libspa-bluetooth`. This looks nicer to me, in theory.

[Third option: For now, simply wait for configuration to stabilize, since things could change?]

Related: #29570
cc @ericonr

### PR DESCRIPTION
**Pull request by st3r4g**
* activate bluetooth support only when the bluetooth plugin is installed
* fix README.voidlinux going in pipewire-doc
* disable SDL2 dep, only used in some examples we don't even ship
* rm INSTALL.msg, configs should be stable now (also mark them in xbps)
* ncursesw.patch is now upstream

Fix #29655

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by st3r4g** [comment]
hmm wait a sec, `pw-cat` disappeared?

File: srcpkgs/pipewire/patches/with-bluetooth.patch [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Activate bluetooth support only when the bluetooth plugin is installed.
+--- src/daemon/media-session.d/media-session.conf.orig	2021-04-06 17:05:05.935876278 +0200
++++ src/daemon/media-session.d/media-session.conf	2021-04-06 17:06:16.599878154 +0200
+@@ -94,12 +94,14 @@
+     with-alsa = [
+         with-audio
+     ]
++    with-bluetooth = [
++        bluez5
++    ]
```
Comment (ericonr) [inline srcpkgs/pipewire/patches/with-bluetooth.patch:10]: Can this be upstreamed? Sounds like a nice improvement overall...

Comment (st3r4g) [reply srcpkgs/pipewire/patches/with-bluetooth.patch:10]: ok I can propose it

File: srcpkgs/pipewire/template [hunk 2]
```diff
@@ -81,6 +84,7 @@ libspa-bluetooth_package() {
 	short_desc+=" - bluetooth plugins"
 	pkg_install() {
 		vmove usr/lib/spa-0.2/bluez5
+		vmove etc/pipewire/media-session.d/with-bluetooth
```
Comment (ericonr) [inline srcpkgs/pipewire/template:87]: Nit: `touch $PKGDESTDIR/etc/...` here and avoid having to deal with the file in two different places?

File: srcpkgs/pipewire/patches/with-bluetooth.patch (hunk 1 above)
Comment (st3r4g) [reply srcpkgs/pipewire/patches/with-bluetooth.patch:10]: hmm I'm not entirely convinced. There are other plugins that do not throw errors even if enabled... I have to ask why. Please don't merge yet

Comment (st3r4g) [reply srcpkgs/pipewire/patches/with-bluetooth.patch]: Upstream decreased the log priority of the message, so it's not an error anymore, and we don't need to change the config. I have backported the patch for this, which is already upstream.

**Pull request merged**